machine_management/cluster_api_machine_management/cluster_api_provider_configurations/cluster-api-config-options-azure.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-api-config-options-azure"]
+= Cluster API configuration options for {azure-full}
+include::_attributes/common-attributes.adoc[]
+:context: cluster-api-config-options-azure
+
+toc::[]
+
+:FeatureName: Managing machines with the Cluster API
+include::snippets/technology-preview.adoc[]
+
+You can change the configuration of your {azure-first} Cluster API machines by updating values in the Cluster API custom resource manifests.
+
+[id="cluster-api-sample-yaml-azure_{context}"]
+== Sample YAML for configuring {azure-full} clusters
+
+The following example YAML files show configurations for an {azure-full} cluster.
```

Comment:
Since {azure-full} resolves to "Microsoft Azure", I think "a" works better than "an".

```suggestion
The following example YAML files show configurations for a {azure-full} cluster.
```

Reply:
great catch! changing to `{azure-short}` so it's just "Azure"